package.json

```
@@ -26,13 +26,23 @@
     "typecheck:shared": "tsc --noEmit --project packages/shared",
     "typecheck:frontend": "tsc --noEmit --project packages/frontend",
     "typecheck:backend": "tsc --noEmit --project packages/backend",
-    "typecheck": "npm run typecheck:shared && npm run typecheck:frontend && npm run typecheck:backend"
+    "typecheck": "npm run typecheck:shared && npm run typecheck:frontend && npm run typecheck:backend",
+    "prepare": "husky install"
   },
   "resolutions": {
     "string-width": "^4.2.0",
     "wrap-ansi": "^7.0.0"
   },
+  "lint-staged": {
+    "*.{js,ts,tsx,svelte}": [
```

```suggestion
    "*.{js,ts,svelte}": [
```

I think in bootc there are no tsx files

---

Correct, removed.